ehsm_kms_service/test/test_aes.py

```diff
@@ -0,0 +1,66 @@
+import requests 
+import json
+
+# Replace <port> with real port number
+port = '3000'
+base_url = "http://127.0.0.1:" + port + "/ehsm?Action="

```

Comment:
why not set another macro for the addr?
ehsm_srv_addr = 'http://127.0.0.1'

base_url =  ehsm_srv_addr +":"+ port + "/ehsm?Action="

Reply:
It will be fix.